sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/AbstractSentryWebFilter.java

```diff
@@ -62,21 +63,23 @@ public AbstractSentryWebFilter(final @NotNull IHub hub) {
 
   protected void doFinally(
       final @NotNull ServerWebExchange serverWebExchange,
-      final @NotNull IHub requestHub,
+      final @NotNull IScopes requestHub,
       final @Nullable ITransaction transaction) {
     if (transaction != null) {
       finishTransaction(serverWebExchange, transaction);
     }
     if (requestHub.isEnabled()) {
+      // TODO close lifecycle token instead of popscope
       requestHub.popScope();
     }
-    Sentry.setCurrentHub(NoOpHub.getInstance());
+    Sentry.setCurrentScopes(NoOpScopes.getInstance());
   }
 
   protected void doFirst(
-      final @NotNull ServerWebExchange serverWebExchange, final @NotNull IHub requestHub) {
+      final @NotNull ServerWebExchange serverWebExchange, final @NotNull IScopes requestHub) {
```

Comment:
Should we rename this to something like `requestScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366